### PR TITLE
Joystick functionality added

### PR DESCRIPTION
**Pull request by SFang-cmd**
Please approve

### CODE REVIEW

**Review by hadley31 — DISMISSED** [review]
lgtm

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -83,15 +84,14 @@ public RobotContainer() {
...
      */
     private void configureButtonBindings() {
-        UserControls controls = new XboxUserControls(0, 1);
+        UserControls controls = new MixedXboxJoystickControls(0, 1, 5);
 
         // Define commands here
         ArcadeDrive defaultDriveCommand = new ArcadeDrive(drive,
                 () -> controls.getLeftDriveX(),
                 () -> controls.getLeftDriveY(),
                 () -> controls.getRightDriveX(),
```
Comment (WAaron-500) [inline src/main/java/frc/robot/RobotContainer.java:102]: I would suggest keeping sniper mode and just make it never called bc I think Matthew implied that he would switch back to xbox if we got working xbox controllers

**Review by WAaron-500 — DISMISSED** [review]
everything else lgtm, but just consider the comment

File: src/main/java/frc/robot/RobotContainer.java (hunk 1 above)
Comment (hadley31) [reply src/main/java/frc/robot/RobotContainer.java:102]: It seems Matthew would rather forfeit the match than stoop as low as using sniper mode.

**Review by hadley31 — APPROVED** [review]

File: src/main/java/frc/robot/RobotContainer.java (hunk 1 above)
Comment (WAaron-500) [reply src/main/java/frc/robot/RobotContainer.java:102]: ok

**Review by WAaron-500 — APPROVED** [review]

**Pull request merged**